modin/pandas/io.py

```diff
@@ -119,17 +124,16 @@ def _read_csv_from_file(filepath, npartitions, kwargs={}):
     """
     empty_pd_df = pandas.read_csv(
         filepath, **dict(kwargs, nrows=0, skipfooter=0, skip_footer=0))
-    names = empty_pd_df.columns
+    column_names = empty_pd_df.columns
 
-    skipfooter = kwargs["skipfooter"]
-    skip_footer = kwargs["skip_footer"]
+    skipfooter = kwargs.get("skipfooter", None)
```

Comment:
For Pandas 0.22:
`skipfooter = kwargs.get("skipfooter", None) or kwargs.get("skip_footer", None)`

If this is Pandas 0.23, it's fine.